docs/operate/mobility/move-arm/configure-additional.md

```diff
@@ -18,25 +18,28 @@ If you have a gripper, camera, or other components attached to your arm, you can
 
 You can also take into account passive objects attached to the arm such as a camera mount to avoid collisions.
 
-## Configure a gripper
+## Configure a gripper's frame
 
 If you have a gripper, you can configure it to move with the arm.
-See [Configure a gripper](/operate/reference/components/gripper/) for instructions on configuring the gripper itself.
-Then, configure the gripper's frame to describe its position and orientation relative to the arm:
+Configure the gripper's frame to describe its position and orientation relative to the arm:
+
+1. See [Configure a gripper](/operate/reference/components/gripper/) for instructions on configuring the gripper itself.
 
 1. In the **CONFIGURE** tab, find the gripper's configuration card.
 
-1. Click **+ Add Frame**.
+1. Click **+ Add frame**.
 
 1. Select the arm's frame as the parent frame.
 
-1. Enter the gripper origin's position and orientation relative to the center of the end of the arm.
-   It is up to you to decide what part of the gripper you want to use as the origin of the frame.
-   It is common to use a central point near the base of the gripper jaws as the origin.
+1. Enter the gripper frame's position and orientation relative to the center of the end of the arm.
+   It is up to you to decide what part of the gripper you want to use as the gripper frame.
+   It can be convenient to use a point near the center of the gripper jaws as the gripper frame.
```

Comment:
This sounds more like guidance. I was reading this wondering how I'd know if that was convenient or not in a specific case
```suggestion
   It is generally convenient to use a point near the center of the gripper jaws as the gripper frame.
```